### PR TITLE
Adds Xybrium Coins to the game

### PR DESCRIPTION
**Pull request by efzapa**
## About The Pull Request

Adds Xybrium Coins to the game, known as zentara, zentari, or for simplicity sake, Platinum.
These are 50 mammon denomination coins meant to fill the hole of merchants often having piles upon piles of gold by the end of the game. Same goes with the king, more often than not they will sometimes deal in big transactions over the max stack size of 200 mammon for gold, so this is an alternative to condense it down, and make it look good at the same time.

(for the sake of lore and me being too lazy to balance a new metal, i am calling Xybrium Unobtainium for now) 

## Why It's Good For The Game

Having large transactions be conducted in a vague amount of coins in a set gold pile sprite that doesnt change from 200 mammons to infinity is kinda dumb. Having working coin piles and sprites for values up to 1000 is very cool.

### CODE REVIEW

**Comment by efzapa** [comment]
> Your atm is missing budget2change() calls.

doesnt need one, i'm not using the shitty alert system

File: code/modules/roguetown/roguemachine/money.dm [hunk 1]
```diff
@@ -85,41 +85,38 @@ GLOBAL_VAR(moneymaster)
...
+				var/zenars = budget/10
+				if(zenars >= 1)
+					for(var/i in 1 to zenars)
+						budget -= 10
+			if("SILVER")
+				var/zenars = budget/5
+				if(zenars >= 1)
+					for(var/i in 1 to zenars)
+						budget -= 5
+			if("BRONZE")
```
Comment (JTGSZ) [inline code/modules/roguetown/roguemachine/money.dm:115]: This man removed the funny bronze effect

Comment (efzapa) [reply code/modules/roguetown/roguemachine/money.dm:115]: does it even work? never seen anyone crushed by the funny effect

**Comment by efzapa** [comment]
takyon 1984'd me

**Pull request closed without merging**